src/build/helpers.ts

```diff
@@ -325,3 +325,8 @@ export function followTypeReferences(
     }
   }
 }
+
+export function uniq(list: string[] = []): string[] {
```

Comment:
`=[]` seems redundant when all callers pass a valid array.

Reply:
Hi @saschanaz, 
Thanks for your comments! I've just started using TS these months, this is my first PR for TS ecosystem XD actually :P.
I updated the codes based on your comment, for the `assertUnique`, didn't use while/for/map for higher performance, but it's easier to understand.

Pls let me know if anything needs to be done. Thanks ;)